pages/Hypr Ecosystem/hyprpaper.md

```diff
@@ -15,10 +15,24 @@ the `wallpaper` keyword is used to apply the preloaded image to your monitor(s):
 
 ```ini
 preload = /home/me/amongus.png
-wallpaper = DP-1,/home/me/amongus.png
+wallpaper = monitor-name,/home/me/amongus.png
```

Comment:
Since that field can accept `desc:` and `contain:` as well, I think a better name would be just `monitor`.